### PR TITLE
[Dictionary] Update exit-repeat

### PR DESCRIPTION
**Pull request by livecodealice**
Put `Form:` on a new line and changed it to `**Form:**` for visibility.
Adjusted indentation of code example and removed empty line.

### CODE REVIEW

File: docs/dictionary/control_st/exit-repeat.lcdoc [hunk 1]
```diff
@@ -34,9 +36,8 @@ condition is false. This example stops the <loop> when the user presses
 the mouse button:
 
     repeat with x = 1 to the number of cards
-    go card x
-    if the mouse is down then exit repeat -- bail out
-
+        go card x
```
Comment (montegoulding) [inline docs/dictionary/control_st/exit-repeat.lcdoc]: @livecodeali @livecodesam should we come up with an example that is moderately useful here? I can't for the life of me think why someone would want to do this...

Comment (montegoulding) [reply docs/dictionary/control_st/exit-repeat.lcdoc]: What about reading from a file in chunks?
```
constant kChunk = 1024
open file tFile for binary read
repeat
    read from file tFile for kChunk
    put the result into tResult
    if tResult is empty or tResult is "eof" then
         ProcessFileChunk it
    end if
    if tResult is not empty then
        exit repeat
    end if
end repeat
close file tFile
```

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 616fb57

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 616fb5710edc6e7f47acd0e21508787c2c1633f6

**Comment by livecode-vulcan** [comment]
😎 test success 616fb5710edc6e7f47acd0e21508787c2c1633f6

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/706)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/722)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/696)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/639)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/642)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/700)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/720)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/705)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/722)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/704)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/715)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/737)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/701)

**Pull request merged**